icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -6085,6 +6086,44 @@ void NumberFormatterApiTest::formatUnitsAliases() {
     }
 }
 
+void NumberFormatterApiTest::TestPortionFormat() {
+    IcuTestErrorCode status(*this, "TestPortionFormat");
+
+    struct TestCase {
+        const char *unitIdentifier;
+        const char *locale;
+        double inputValue;
+        UnicodeString expectedOutput;
+    } testCases[]{
+        {"portion-per-1e9", "en-US", 1, "1 part per billion"},
+        {"portion-per-1e9", "en-US", 2, "2 parts per billion"},
+        {"portion-per-1e9", "en-US", 1000000, "1,000,000 parts per billion"},
+        {"portion-per-1e9", "de-DE", 1000000, "1.000.000 Milliardstel"},
+        {"portion-per-1e1", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e2", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e3", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e4", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e5", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e6", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e7", "en-US", 1, "UNKNOWN"},
+        {"portion-per-1e8", "en-US", 1, "UNKNOWN"},
```

Comment:
All of these `UNKNOWN` things are being caught by the `logKnownIssue()` below, right?  You should probably put comments on the test cases themselves indicating this too.

Reply:
yes, all of them are being caught  by `logKnownIssue`, 

Okay, I will add a comment in a following PR.